.code-samples.meilisearch.yaml

```diff
@@ -382,35 +380,38 @@ faceted_search_update_settings_1: |-
   curl \
     -X POST 'http://localhost:7700/indexes/movies/settings' \
     --data '{
-        "attributesForFaceting": [
+        "filterableAttributes": [
             "director",
             "genres"
         ]
     }'
-faceted_search_facet_filters_1: |-
+faceted_search_filter_1: |-
   curl 'http://localhost:7700/indexes/movies/search' \
-  --data '{ "q": "thriller", "facetFilters": [["genres:Horror", "genres:Mystery"], "director:Jordan Peele"] }'
+  --data '{ "q": "thriller", "filter": [["genres:Horror", "genres:Mystery"], "director:Jordan Peele"] }'
 faceted_search_facets_distribution_1: |-
   curl --get 'http://localhost:7700/indexes/movies/search' \
     --data '{ "q": "Batman", "facetsDistribution": ["genres"] }'
-faceted_search_walkthrough_attributes_for_faceting_1: |-
+faceted_search_walkthrough_filterable_attributes_1: |-
   curl \
     -X POST 'http://localhost:7700/indexes/movies/settings' \
     --data '{
-        "attributesForFaceting": [
+        "filterableAttributes": [
             "director",
             "producer",
             "genres",
             "production_companies"
         ]
     }'
-faceted_search_walkthrough_facet_filters_1: |-
+faceted_search_walkthrough_filter_1: |-
   curl 'http://localhost:7700/indexes/movies/search' \
-  --data '{ "q": "thriller", "facetFilters": [["genres:Horror", "genres:Mystery"], "director:Jordan Peele"] }'
+  --data '{ "q": "thriller", "filter": [["genres:Horror", "genres:Mystery"], "director:Jordan Peele"] }'
 faceted_search_walkthrough_facets_distribution_1: |-
   curl 'http://localhost:7700/indexes/movies/search' \
   --data '{  "q": "Batman", "facetsDistribution": ["genres"] }'
 post_dump_1: |-
   curl -X POST 'http://localhost:7700/dumps'
 get_dump_status_1: |-
   curl -X GET 'http://localhost:7700/dumps/20201101-110357260/status'
+phrase_search_1: |-
+  curl -X POST 'http://localhost:7700/indexes/movies/search' \
+  --data '{ "q": "\"neo\"" }'
```

Comment:
I'm not sure `neo` is a good choice here. The rationale you described over Slack is good (`neo` vs `neo-kantian`, `neo-marxist`, `neo-*`) but I'm getting quite a few results with that query. I believe that's because `-` is a soft separator and thus, for what we're trying to demonstrate here, functionally the same as whitespace.

Reply:
Perhaps proper names are a better choice? `John Cusack` will have loads of results because John is such a common name, whereas `"John Cusack"` will only give you documents with that specific actor.

Reply:
Ah, got it. Shame to miss the opportunity for a Neon Genesis Evangelion reference 😭 
I still think common prefixes are a great way to demonstrate this feature, since you can use phrase search to get rid of the noise from our prefix search. Maybe something like `Up` would be a good example query? If you wanted to find the Pixar movie "Up" without getting results for "upper", "upside", "uphold", etc.

Reply:
We will get an Evangelion reference there one day, don't worry.

As for the term, `up` does illustrate the point. It will still give you loads of results because it's such a common word, but it's definitely in line with what phrase search does.

Reply:
Another decent example category for this would be words or names that are very close (one typo away) to other more common words. For example if you queried "prophete" looking for the movie "un prophète" and you were inundated with results containing the English word "prophet", you might solve the problem w/ a phrase search (not sure if MeiliSearch would actually consider "prophete" and "prophet" equivalent for that query).

Reply:
Actually, multilingual datasets provide a lot of good examples in this area.

Reply:
it could also be interesting to add both a word between quotes `""` and a normal word. As this looks like unnecessary quotes and not a phrase search

for example `big "panda"`

Reply:
Also, `neo` alone acts the same as `"neo"` since it is to small of a word to accept any typo! 😅

Reply:
@bidoubiwa We decided to go with `"african american" horror` as the example, finding a Jordan Peele movie.